src/main/java/com/SpringBootProject/DaoServices/ClientsDaoService.java

```diff
@@ -0,0 +1,187 @@
+package com.SpringBootProject.DaoServices;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import com.SpringBootProject.Entity.Clients;
+
+
+public class ClientsDaoService implements ClientsDaoInterface {
+	private static String URL = "jdbc:mysql://localhost:3306/TestProject";
```

Comment:
Delete "application.properties" file if you don't use it, or read values from this file and use them in database connection.